trin-history/src/lib.rs

```diff
@@ -1,37 +1,49 @@
-use std::env;
-
 use log::info;
+use serde_json::Value;
 use tokio::sync::mpsc;
 
 use trin_core::cli::TrinConfig;
-use trin_core::jsonrpc::launch_trin;
 use trin_core::portalnet::protocol::{
-    JsonRpcHandler, PortalEndpoint, PortalnetConfig, PortalnetProtocol,
+    HistoryEndpointKind, HistoryNetworkEndpoint, PortalnetConfig, PortalnetProtocol,
 };
 
+pub struct HistoryRequestHandler {
+    pub history_rx: mpsc::UnboundedReceiver<HistoryNetworkEndpoint>,
+}
+
+impl HistoryRequestHandler {
+    pub async fn process_network_requests(mut self) {
```

Comment:
"network" requests makes me think peer to peer. But if I'm reading this right, it's all about serving the json-rpc requests, yeah? Another option is something like:
```suggestion
    pub async fn handle_client_queries(mut self) {
```

Reply:
In my head it was something along the lines of "process these requests (from the jsonrpc handler) for data from your (sub)network". But, I like your suggestion and updated it. The fewer times we use "network" the better.